test/index.js

```diff
@@ -1955,4 +1956,35 @@ describe('Toys', () => {
             expect(Toys.asyncStorageInternals()).to.be.an.instanceof(Map);
         });
     });
+
+    describe('joi patch schema', () => {
```

Comment:
I think we need to add some tests that proves `patchJoiSchema()` does what it purports to do.  For example, we should provide it a schema with usage of `required()` and `default()`, and confirm that the resulting schema doesn't apply defaults or required fields.

Reply:
yes, I think we need to but I wasn't sure how to do this. Are there convenience methods in `Joi` to do such assertions?

Reply:
okay, found `.describe()` which may come in handy!